src/libraries/System.Diagnostics.Process/tests/ProcessModuleTests.Windows.cs

```diff
@@ -36,9 +36,7 @@ public void LongModuleFileNamesAreSupported()
             IntPtr moduleHandle = Interop.Kernel32.LoadLibrary(longNamePath);
             if (moduleHandle == IntPtr.Zero)
             {
-                Assert.Equal(126, Marshal.GetLastWin32Error()); // ERROR_MOD_NOT_FOUND
-                Assert.Equal(Architecture.X86, RuntimeInformation.ProcessArchitecture);
```

Comment:
Should we keep this assert or is it unstable as well?

Reply:
I was hoping that 126 (ERROR_MOD_NOT_FOUND) is the only error we can get from `LoadLibrary`, but from the [log](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-57551-merge-0bad4529cfd6489ebd/System.Diagnostics.Process.Tests/1/console.176badb8.log?sv=2019-07-07&se=2021-09-06T17:02:55Z&sr=c&sp=rl&sig=kky7AFZZa%2FGBIEFSw64gnh3dLNJy0djiLq89u5T1tSU%3D) that you have provided it seems that this syscall can fail with other errors as well. 

FWIW in this particular case it failed with `ERROR_INSUFFICIENT_BUFFER` (The data area passed to a system call is too small) while we don't pass any buffers to the syscall (assuming that we don't threat the valid path as buffer)

Reply:
Just to clarify, I was referring to the Assert from line 40